### PR TITLE
Fix y share

### PR DESCRIPTION
**Pull request by lakhoune**
This PR fixes the shared data types used by syncmeta. 
It seems that since upgrading to the latest yjs version, `y.share.foo` can no longer be used to get/create a shared data type `foo`
We therefore need to transform every `y.share.foo` into `y.getMap('foo')` or `y.getText('foo')`, ... depending on the data type

### CODE REVIEW

File: app/src/static-app.js [hunk 1]
```diff
@@ -207,7 +207,8 @@ class StaticApp extends PolymerElement {
     this.publishUpdateMetamodelOperation();
     this.changeVisibility("#generateModelLoader", true);
     this.initY((y) => {
-      y.share.metamodelStatus.observe((event) => {
+      const metaModelStatus = y.getMap("metaModelStatus");
```
[the remark this answers is not in the document]
Comment (lakhoune) [reply app/src/static-app.js:210]: I did not even pay attention to that. But as long as every change of `metamodelStatus` to `metaModelStatus` was done then I think we should stick with that as our naming convention is camelCase.

[the remark this answers is not in the document]
Comment (lakhoune) [reply app/src/static-app.js:210]: Though you could argue that metamodel is one word and therefore model should not be capitalized

File: widgets/src/js/activity_widget.js [hunk 2]
```diff
@@ -3,45 +3,55 @@
...
-                for (var i = 0; i < userList.length; i++) {
-                    activtyList.addUser(userList[i]);
-                }
-                activtyList.init();
-            });
+      WaitForCanvas(CONFIG.WIDGET.NAME.ACTIVITY, 7).done(function (data) {
+        console.info("ACTIVITY: Got message from CANVAS");
+        var user = data.local.user;
+        const usersMap = y.getMap("users");
+        const userList = y.getMap("userList");
```
[the remark this answers is not in the document]
Comment (lakhoune) [reply widgets/src/js/activity_widget.js:39]: Yes this is a mistake that I need to fix

[the remark this answers is not in the document]
Comment (lakhoune) [reply widgets/src/js/activity_widget.js:39]: Ill stick with `userList` since that was the original name

**Review by phil-cd — APPROVED** [review]

**Pull request merged**